pages/instructorAssessmentManualGrading/instructorAssessmentManualGrading.sql

```diff
@@ -0,0 +1,25 @@
+-- BLOCK select_instance_questions_manual_grading
+SELECT
+    iq.*,
+    s.graded_at,
+    q.id AS question_id,
+    aq.max_points,
+    ai.id AS assessment_instance_id,
+    qo.question_number
+FROM
+    instance_questions AS iq
+    JOIN assessment_instances AS ai ON (ai.id = iq.assessment_instance_id)
+    JOIN assessment_questions AS aq ON (aq.id = iq.assessment_question_id)
+    JOIN questions AS q ON (q.id = aq.question_id)
+    JOIN question_order(ai.id) AS qo ON (qo.instance_question_id = iq.id)
+    JOIN variants AS v ON (v.instance_question_id = iq.id)
+    JOIN (
+        -- We only want the last submission...
+        SELECT * FROM submissions AS s
```

Comment:
When we have a question that's been graded, and then there's a new variant generated and saved, this seems to have some issues. The score displayed belongs to the graded question, but the grading page brings you to the saved submission.

Reply:
Ah, if I understand correctly, do you want the grading page to bring you to the graded submission instead? 

Just looking at this briefly, do we perhaps want the last graded_at submission? AKA. The last submission that graded_at != null?